src/runtime-rs/crates/hypervisor/ch-config/src/lib.rs

```diff
@@ -479,3 +481,15 @@ fn usize_is_zero(v: &usize) -> bool {
 fn u16_is_zero(v: &u16) -> bool {
     *v == 0
 }
+
+// Type used to simplify conversion from a generic Hypervisor config
+// to a CH specific VmConfig.
+#[derive(Debug, Clone)]
+pub struct NamedHypervisorConfig {
+    pub kernel_params: String,
+    pub sandbox_path: String,
+    pub vsock_socket_path: String,
+    pub cfg: HypervisorConfig,
+
```

Comment:
thanks @jodh-intel , if there's not reason to don't save space here, could you remove this blank line?

Reply:
Good catch @dborquez! @amshinde or I will remove that in the next PR :smile: